src/main/java/dev/ftb/packcompanion/mixin/FTBPCMixinPlugin.java

```diff
@@ -0,0 +1,39 @@
+package dev.ftb.packcompanion.mixin;
+
+import org.objectweb.asm.tree.ClassNode;
+import org.spongepowered.asm.mixin.extensibility.IMixinConfigPlugin;
+import org.spongepowered.asm.mixin.extensibility.IMixinInfo;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Mixin config plugin for FTB Pack Companion.
+ * Conditionally applies JER mixin only when JustEnoughResources is present.
+ */
+public class FTBPCMixinPlugin implements IMixinConfigPlugin {
+
+    private static final String JER_BIOME_HELPER_MIXIN = "dev.ftb.packcompanion.mixin.features.jerfix.BiomeHelperMixin";
+    private boolean jerLoaded;
+
+    @Override
+    public void onLoad(String mixinPackage) {
+        // Check if JER is present by looking for the class file without loading it
+        jerLoaded = getClass().getClassLoader().getResource("jeresources/api/util/BiomeHelper.class") != null;
```

Comment:
Can we not simply do `ModList.get().isLoaded("jer")` in `shouldApplyMixin()`?  (note: not sure of the mod ID for JER, double check that). No need for a dependency here, even compile-time.

Reply:
It's also worth noting that you do not need to load the mixin based on if the mod is present. Nor implement a plugin to do this. You can simply add `@Pseudo` to the class and it will silence any errors triggered by the targetting class being missing at runtime.

Reply:
Yeah go with `@Pseudo`.  And use backticks to avoid tagging random people on github :laughing: